pages/common/tr.md

```diff
@@ -1,31 +1,27 @@
 # tr
 
-> Translate characters: run replacements based on single characters and character sets.
+> Translate characters: the tr utility will translate (i.e., substitute or delete) charaters from an input stream and output those results.
 
-- Replace all occurrences of a character in a file, and print the result:
+- Replace all occurrences of a character in a file, and print the result on the screen (file remains unchanged):
 
 `tr {{find_character}} {{replace_character}} < {{filename}}`
 
 - Replace all occurrences of a character from another command's output:
 
 `echo {{text}} | tr {{find_character}} {{replace_character}}`
 
-- Map each character of the first set to the corresponding character of the second set:
```

Comment:
Why is this example removed?

Reply:
I think it's misleading and doesn't work the way it reads. Plus, not necessary to get the point of tr across. I'll leave it.